_includes/site/footer.html

```diff
@@ -61,19 +61,19 @@ <h3>News feeds</h3>
 
     <div class="link-item">
       <a class="link-item__main" href="/feed.xml" rel="alternate" type="application/atom+xml">
-        {%- include icons/rss.svg %}
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><use href="#icon-rss"></use></svg>
```

Comment:
I fail to see the improvement here 🤔

Reply:
Less duplication in the rendered HTML. Which included a duplicate `id`.
I suppose we could also remove the `<use>` references alltogether and always use an include without `id` on the `<svg>` element.